idf_build_apps/__init__.py

```diff
@@ -12,7 +12,12 @@
 
 __version__ = '2.0.0b3'
 
+from .session_args import (
+    SessionArgs,
+)
+
 LOGGER = logging.getLogger('idf_build_apps')
+SESSION_ARGS = SessionArgs()
```

Comment:
```suggestion
LOGGER = logging.getLogger('idf_build_apps')

from .session_args import (
    SessionArgs,
)

SESSION_ARGS = SessionArgs()
```

this could help workaround the `from . import LOGGER` change you did in the `utils.py`

Reply:
Good! It has run successfully for me.